docs/docs/_available-tags.md

```diff
@@ -0,0 +1,21 @@
+# Available tags
+
+The following is a list of available tags that can be used when contributing a sample. Click on the links to view the samples associated with the tags.
+
+Tag | Description
+-- | --
+[AI](./tags#AI) | Content dealing with AI related subjects. 
+[Cleanup](./tags#Cleanup) | Content about removing things and cleaning up.
+[Files](./tags#Files) | Content about dealing with files.
+[Guests](./tags#Guests) | Content about working with external users.
+[Lists](./tags#Lists) | Content involving lists or libraries.
+[Migration](./tags#Migration) | Content about exporting, importing, syncing or moving data.
+[Failures](./tags#Failures) | Content dealing with problems and failures. 
+[Privacy](./tags#Privacy) | Content about privacy related subjects.
+[Provisioning](./tags#Provisioning) | Content about creating structures or content programmatically.
+[Reports](./tags#Reports) | Content about getting lists of information for reporting purposes.
+[Security](./tags#Security) | Content about security and permissions.
```

Comment:
Permissions are a part of security, so we should remove `security` from the description, because it's repeating the tag, without explaining what we actually mean.